experimental/PlaneCurve/ParaPlaneCurves.jl

```diff
@@ -0,0 +1,152 @@
+export parametrization_plane_curve, adjoint_ideal, rational_point_conic,
+       parametrization_conic, map_to_rational_normal_curve,
+       rat_normal_curve_anticanonical_map, rat_normal_curve_It_Proj_Odd,
+       rat_normal_curve_It_Proj_Even
+
+
+################################################################################
+function _tosingular(C::ProjPlaneCurve{fmpq})
+    F = C.eq
+    T = parent(F)
+    Tx = singular_ring(T)
+    return Tx(F)
+end
+
+function _fromsingular_ring(R::Singular.PolyRing)
```

Comment:
Just a minor comment for if we ever mainstream `fromsingular_ring`. The variables that we put into a singular polynomial ring may not be the same as the ones that you get out #574. Therefore, maybe this function should have accept a list of strings for variable names. This is not for now, just something to keep on your radar.